src/DssSpell.t.sol

```diff
@@ -528,81 +508,86 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(vest.fin(8), block.timestamp);
     }
 
-    function testVestMKR() private { // make public to use
+    function testVestMKR() public {
         VestAbstract vest = VestAbstract(addr.addr("MCD_VEST_MKR_TREASURY"));
-        assertEq(vest.ids(), 23);
+        assertEq(vest.ids(), 24);
 
         uint256 prevAllowance = gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY"));
 
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + 803 * WAD);
+        assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + 787.70 ether);
 
         assertEq(vest.cap(), 1_100 * WAD / 365 days);
-        assertEq(vest.ids(), 24);
+        assertEq(vest.ids(), 28);
 
-        // Friday,   1 July 2022 00:00:00 UTC
-        uint256 JUL_01_2022 = 1656633600;
-        // Saturday, 1 July 2023 00:00:00 UTC
-        uint256 JUL_01_2023 = 1688169600;
+        uint256 AUG_01_2022 = 1659326400;
+        uint256 AUG_01_2023 = 1690862400;
+        uint256 SEP_28_2022 = 1664337600;
+        uint256 SEP_28_2024 = 1727496000;
 
-        address GRO_WALLET = wallets.addr("GRO_WALLET");
+        address GOV_WALLET1 = 0xbfDD0E744723192f7880493b66501253C34e1241;
+        address GOV_WALLET2 = 0xbb147E995c9f159b93Da683dCa7893d6157199B9;
+        address GOV_WALLET3 = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
```

Comment:
consider adding those addrs to `wallets`

Reply:
I don't think we normally add these types of wallets. For example PE has direct wallets listed for MKR vests, but they aren't in here.

Reply:
yeah, we got some vest wallets there though

Reply:
as discussed in chat, we could include a wallet list for mkr streams and other eventually to monitor things like allowance and make sure they're within the expected range, to be addressed in a separate pr eventually, going to resolve for now.